examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,101 @@
+"""
+============================
+TickedStroke Patheffect Demo
+============================
+
+Matplotlib's :mod:`.patheffects` can be used to alter the way paths
+are drawn at a low enough level that they can affect almost anything.
+
+The :doc:`patheffects guide</tutorials/advanced/patheffects_guide>`
+details the use of patheffects.
+
+The `~matplotlib.patheffects.TickedStroke` patheffect illustrated here
+draws a path with a ticked style.  The spacing, length, and angle of
+ticks can be controlled.
+
+See also the :doc:`contour demo example
+</gallery/lines_bars_and_markers/lines_with_ticks_demo>`.
+
+See also the :doc:`contours in optimization example
+</gallery/images_contours_and_fields/contours_in_optimization_demo>`.
+"""
+
+import matplotlib.patches as patches
+from matplotlib.path import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as PathEffects
+
+# Direct to path
+fig, ax1 = plt.subplots(figsize=(6, 6))
+path = Path.unit_circle()
+patch = patches.PathPatch(path, facecolor='none', lw=2, path_effects=[
+    PathEffects.withTickedStroke(angle=90, spacing=10, length=1, flip=True)])
+
+ax1.add_patch(patch)
+ax1.axis('equal')
+ax1.set_xlim(-2, 2)
+ax1.set_ylim(-2, 2)
+
+plt.show()
+
+###############################################################################
+# Lines and curves with plot and legend
+fig, ax3 = plt.subplots(figsize=(6, 6))
+p1, = ax3.plot([0, 1], [0, 1],
+               path_effects=[PathEffects.withTickedStroke(spacing=7,
+                                                          angle=135)])
+
+nx = 101
+x = np.linspace(0.0, 1.0, nx)
+y = 0.3*np.sin(x*8) + 0.4
+p2, = ax3.plot(x, y, path_effects=[PathEffects.withTickedStroke()])
+
+leg = ax3.legend([p1, p2], ["Line", "Curve"], fancybox=True,
+                 loc='upper left', edgecolor='k')
+
+plt.show()
+
+###############################################################################
+# Contour plot with objective and constraints.
+# Curves generated by contour to represent a typical constraint in an
+# optimization problem should be plotted with angles between zero and
+# 180 degrees.
+fig, ax4 = plt.subplots(figsize=(6, 6))
+
+nx = 101
+ny = 105
+
+# Set up survey vectors
+xvec = np.linspace(0.001, 4.0, nx)
+yvec = np.linspace(0.001, 4.0, ny)
+
+# Set up survey matrices.  Design disk loading and gear ratio.
+x1, x2 = np.meshgrid(xvec, yvec)
+
+# Evaluate some stuff to plot
+obj = x1**2+x2**2-2*x1-2*x2+2
+g1 = -(3*x1+x2-5.5)
+g2 = -(x1+2*x2-4)
+g3 = .8+1/(x1**3)-x2
+
+cntr = plt.contour(x1, x2, obj, [0.01, 0.1, 0.5, 1, 2, 4, 8, 16],
```

Comment:
Use the `Axes` variable (`ax4` or `ax3` if renumbered) instead of `plt`.

Reply:
Fixed in 9a8a4389b below.